website/docs/r/frontdoor.html.markdown

```diff
@@ -255,9 +255,11 @@ The following attributes are only valid if `certificate_source` is set to `Azure
 
 ## Attributes Reference
 
-`backend_pool` exports the following:
-
-* `id` - The ID of the Azure Front Door Backend Pool.
+* `backend_pool_health_probes` - A map/dictionary of Backend Pool Health Probe Names (key) to the Backend Pool Health Probe ID (value)
+* `backend_pool_load_balancing_settings` - A map/dictionary of Backend Pool Load Balancing Setting Names (key) to the Backend Pool Load Balancing Setting ID (value)
+* `backend_pools` - A map/dictionary of Backend Pool Names (key) to the Backend Pool ID (value)
+* `frontend_endpoints` - A map/dictionary of Frontend Endpoint Names (key) to the Frontend Endpoint ID (value)
+* `routing_rules` - A map/dictionary of Routing Rule Names (key) to the Routing Rule ID (value)
 
```

Comment:
Can we add a blurb at the top of this documentation the calls out the breaking change and why it is required? Maybe something like:

-> **BREAKING CHANGE:** In order to address the ordering issue we have changed the design on how to retrieve existing sub resources such as frontend endpoints, routing rules, etc. Please refer to the updated documentation below for more information.